StortSpelProjekt/Engine/src/Renderer/Transform.h

```diff
@@ -46,6 +46,8 @@ class Transform
 	DirectX::XMFLOAT3 GetScale() const;
 	// gets the rotation of the transform in all axisis
 	DirectX::XMMATRIX GetRotMatrix() const;
+	DirectX::XMFLOAT3 GetForwardVec() const;
+	DirectX::XMFLOAT3 GetUpVec() const;
```

Comment:
These functions have been added in develop from another branch, make sure there are not duplicates when you merge